common/constants/default.go

```diff
@@ -0,0 +1,16 @@
+package constants
+
+// Resources
+const (
+	MY_NODE_NAME = "MY_NODE_NAME"
+	ServerDefaultACLDirectory = "/etc/kubeedge/edgemesh/server/acls"
+	AgentDefaultACLDirectory  = "/etc/kubeedge/edgemesh/agent/acls"
+	ServerDefaultKeyFile      = ServerDefaultACLDirectory + "/server.key"
+	AgentDefaultKeyFile       = AgentDefaultACLDirectory + "/server.key"
+
+	SECRET_NAMESPACE  = "kubeedge"
+	SECRET_NAME       = "edgemeshaddrsecret"
+	SERVER_ADDR_NAME  = "edgemeshserver"
+	CONNECTION_CLOSED = "use of closed network connection"
```

Comment:
The naming format of the constant should be consistent， use `SecertNamespace` rather than `SECRET_NAMESPACE`

Reply:
ok, already modified